.fullsend/scripts/validate-output-schema.sh

```
@@ -0,0 +1,54 @@
+
+with open(sys.argv[1]) as f:
+    instance = json.load(f)
+with open(sys.argv[2]) as f:
+    schema = json.load(f)
+try:
+    validate(instance=instance, schema=schema)
+    print('PASS: output validated against schema')
+except ValidationError as e:
+    print(f'FAIL: schema validation error: {e.message}')
+    if e.path:
+        print(f'  at: {".".join(str(p) for p in e.path)}')
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)